code/API_definitions/NetworkSliceBooking.yaml

```diff
@@ -0,0 +1,508 @@
+      description: Correlation id for the different services
+      schema:
+        type: string
+  schemas:
+    SessionId:
+      description: Session ID in UUID format
+      type: string
+      format: uuid
+
+    SessionInfo:
+      description: Session related information.
+      allOf:
+        - $ref: "#/components/schemas/CreateSession"
+        - type: object
+          properties:
+            sessionId:
+              $ref: "#/components/schemas/SessionId"
+            result:
+              type: string
+              enum:
+                - Success
+                - Fail
+          required:
+            - result
+
+
+
+    CreateSession:
+      description: Attributes required to create a session
+      type: object
+      properties:
+        ServiceTime:
+          $ref: "#/components/schemas/ServiceTime"
+        ServiceArea:
+          $ref: "#/components/schemas/Area"
+        QoSProfile:
+          $ref: "#/components/schemas/QoSProfile"
+      required:
+        - ServiceTime
+        - ServiceArea
+        - QoSProfile
+
+    QoSProfile:
+      type: object
+      properties:
+        MaxNumofTerminals:
+          $ref: "#/components/schemas/NumberOfTerminals"
+        DLThroughputPerTerminal:
+          $ref: "#/components/schemas/Rate"
+        ULThroughputPerTerminal:
+          $ref: "#/components/schemas/Rate"
+        DLLatency:
+          description: |
+            DLLatency is an attribute specifies the required DL packet transmission latency (millisecond) through the 5G network.
+          $ref: "#/components/schemas/Duration"
+        ULLatency:
+          description: |
+            ULLatency is an attribute specifies the required UL packet transmission latency (millisecond) through the 5G network.
+          $ref: "#/components/schemas/Duration"
+
+
+    Duration:
+      description: Specification of duration
+      type: object
+      properties:
+        value:
+          description: Quantity of duration
+          type: integer
+          example: 12
+          format: int32
+          minimum: 1
+        unit:
+          $ref: "#/components/schemas/TimeUnitEnum"
+
+
+    TimeUnitEnum:
+      description: Units of time
+      type: string
+      enum:
+        - Days
+        - Hours
+        - Minutes
+        - Seconds
+        - Milliseconds
+        - Microseconds
+        - Nanoseconds
+
+
+    Rate:
+      description: Specification of rate
+      type: object
+      properties:
+        value:
+          description: Quantity of rate
+          type: integer
+          example: 10
+          format: int32
+          minimum: 0
+          maximum: 1024
+        unit:
+          $ref: "#/components/schemas/RateUnitEnum"
+
+    RateUnitEnum:
+      description: Units of rate
+      type: string
+      enum:
+        - bps
+        - kbps
+        - Mbps
+        - Gbps
+        - Tbps
+
+    NumberOfTerminals:
+      type: integer
+      example: 5
+      description: Number of terminals

```

Comment:
The term 'device' is used in other CAMARA APIs and I assume, that a 'terminal" is the same as a 'device'. When there is indeed no difference between 'terminal' and 'device', I suggest to use change to 'MaxNumberOfDevices' (or MaximumNumberOfDevices)..

Reply:
> The term 'device' is used in other CAMARA APIs and I assume, that a 'terminal" is the same as a 'device'. When there is indeed no difference between 'terminal' and 'device', I suggest to use change to 'MaxNumberOfDevices' (or MaximumNumberOfDevices)..

That's okay for me. Fan and me will modify this. For further comments, I'll suggest to initiate another issue and pr to keep updating the yaml.